inductiva/tests/simulators/test_dummy_simulator.py

```diff
@@ -0,0 +1,89 @@
+"""Tests on a dummy simulator for testing purposes."""
+from typing import Optional, List
+import tempfile
+import json
+import os
+
+from inductiva import simulators, tasks, types
+
+INPUT_DIR = os.path.join(os.path.dirname(__file__), "test_input_dir")
+BACKEND_ARGS_FILE = "test_arguments.json"
+TEMP_DIR = tempfile.TemporaryDirectory()
+
+class DummySimulator(simulators.Simulator):
```

Comment:
I think we can move this class to the main package. The user can use it to test the installation.

Reply:
Alrighty!